### PR TITLE
change make to build

### PR DESCRIPTION
**Pull request by PetervDooren**

### CODE REVIEW

File: install.bash [hunk 1]
```diff
@@ -19,6 +19,12 @@ then
     sudo apt-get install -y -q python-is-python3
 fi
 
+if ! dpkg -s python3-catkin-tools &> /dev/null
```
Comment (MatthijsBurgh) [inline install.bash:22]: You need ROS apt sources for this. Or install via pip.

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
This will produce too much log lines in CI, need to use `--no-status` in CI.

File: setup/emc-update-impl.bash [hunk 2]
```diff
@@ -124,7 +124,7 @@ fi
 _make_sure_installed python3-catkin-tools libassimp-dev ros-"${EMC_ROS_DISTRO}"-cv-bridge ros-"${EMC_ROS_DISTRO}"-image-geometry ros-"${EMC_ROS_DISTRO}"-map-server ros-"${EMC_ROS_DISTRO}"-message-generation ros-"${EMC_ROS_DISTRO}"-message-runtime ros-"${EMC_ROS_DISTRO}"-nav-msgs ros-"${EMC_ROS_DISTRO}"-roscpp ros-"${EMC_ROS_DISTRO}"-shape-msgs ros-"${EMC_ROS_DISTRO}"-tf2
 
 # 4) Compile
-catkin build --workspace "$EMC_SYSTEM_DIR"
+catkin build --workspace "$EMC_SYSTEM_DIR" --no-status
```
Comment (MatthijsBurgh) [inline setup/emc-update-impl.bash:127]: I do think you want the status outside CI. So this should be conditional based on CI env variabele.

**Comment by MatthijsBurgh** [comment]
CI doesn't fail on the build failing.

**Comment by MatthijsBurgh** [comment]
Sadly `tf` is still needed because of `tf` conversions in geolib. Getting rid of this requires migrating the users of these to migrate to `tf2` conversions.

**Comment by PetervDooren** [comment]
Mad issue https://github.com/tue-robotics/emc-env/issues/28 for CI not failing on a failed build

**Review by MatthijsBurgh — APPROVED** [review]
CI bugs will be fixed in another branch

**Pull request merged**